rust-toolchain.toml

```diff
@@ -1,3 +1,3 @@
 [toolchain]
-channel = "1.87.0"
+channel = "stable"
```

Comment:
To make the build reproducible, I feel we should pin to a specific version of Rust. Otherwise, it may break accidentally due to Rust upgrade or we may not reproduce the previously build results